server/src/graphql/mutations/createApp.ts

```diff
@@ -28,17 +31,56 @@ export const createApp: MutationResolvers['createApp'] = async (
   if (isAppNameTaken) {
     throw new Error('App name already taken');
   }
+  // we do git related operations only for apps
+  // that have gitRepoURl
+  let repo;
+  if (input.gitRepoUrl) {
+    const repoData = getRepoData(input.gitRepoUrl);
+    try {
+      const res = await fetch(
+        `https://api.github.com/repos/${repoData.owner}/${repoData.repoName}`
+      );
+      repo = await res.json();
+    } catch (error) {
+      console.log(error);
+    }
+  }
```

Comment:
Should we add a check to verify that the user own the repo for now? You should not be allowed to clone one of my public repo I think.

Reply:
@pradel for this one I do not think that there's any benefit for having that check. 

Multiple cases come in mind for this : 
- just want to test with some generic public repo w/o forking 
- login from one github acc and want to test something from some gh org that is  public

Reply:
Resolved as this will be done when fixing the : https://github.com/ledokku/ledokku/issues/295